holmes/config.py

```diff
@@ -477,8 +479,8 @@ def _get_llm(self, model_key: Optional[str] = None, tracer=None) -> "LLM":
             )
             api_key = model_params.pop("api_key", api_key)
             model = model_params.pop("model", model)
-
-        from holmes.core.llm import DefaultLLM
+        if not api_key and "robusta.dev" in model_params.get("base_url", "") and self.api_key:
```

Comment:
Thinking here loud, can't we just add it to the robusta model in the model_list when loading it and keep this logic generic? I think it will be good idea going forward. WDYT?

Also just making sure did you validate the cli and make sure it isn't breaking anything?

Reply:
@moshemorad do you mean adding the robusta ai specific code to load_model_from_file ?
It seems the same to me